tests/components/logbook/test_init.py

```diff
@@ -434,6 +457,63 @@ def test_include_events_domain(self):
             entries[2], pointB, "blu", domain="sensor", entity_id=entity_id2
         )
 
+    def test_include_events_domain_glob(self):
+        """Test if events are filtered if domain or glob is included in config."""
+        assert setup_component(self.hass, "alexa", {})
+        entity_id = "switch.bla"
+        entity_id2 = "sensor.blu"
+        entity_id3 = "switch.included"
+        pointA = dt_util.utcnow()
+        pointB = pointA + timedelta(minutes=logbook.GROUP_BY_MINUTES)
+        pointC = pointB + timedelta(minutes=logbook.GROUP_BY_MINUTES)
+        entity_attr_cache = logbook.EntityAttributeCache(self.hass)
+
+        event_alexa = MockLazyEventPartialState(
+            EVENT_ALEXA_SMART_HOME,
+            {"request": {"namespace": "Alexa.Discovery", "name": "Discover"}},
+        )
+
+        eventA = self.create_state_changed_event(pointA, entity_id, 10)
+        eventB = self.create_state_changed_event(pointB, entity_id2, 20)
+        eventC = self.create_state_changed_event(pointC, entity_id3, 30)
+
+        config = logbook.CONFIG_SCHEMA(
+            {
+                ha.DOMAIN: {},
+                logbook.DOMAIN: {
+                    logbook.CONF_INCLUDE: {
+                        CONF_DOMAINS: ["homeassistant", "sensor", "alexa"],
+                        CONF_ENTITY_GLOBS: ["*.included"],
+                    }
+                },
+            }
+        )
+        entities_filter = convert_include_exclude_filter(config[logbook.DOMAIN])
+        events = [
+            e
+            for e in (
+                MockLazyEventPartialState(EVENT_HOMEASSISTANT_START),
+                event_alexa,
+                eventA,
+                eventB,
+                eventC,
+            )
+            if logbook._keep_event(self.hass, e, entities_filter, entity_attr_cache)
+        ]
+        entries = list(logbook.humanify(self.hass, events, entity_attr_cache))
+
+        assert len(entries) == 4
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5745133)